src/main/resources/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration/help-jwt.html

```diff
@@ -0,0 +1,3 @@
+<div>
```

Comment:
I just realised the help for the URL field for Jira Server is outdated now that we have Jira Cloud support:
<img width="1283" alt="Screenshot 2019-04-05 at 11 00 42" src="https://user-images.githubusercontent.com/908592/55620297-774eb780-5792-11e9-8efa-6c80fea428a6.png">

Would be nice if you could update it and use a similar template with the link to our documentation.